src/java/org/apache/cassandra/db/guardrails/Guardrails.java

```diff
@@ -315,6 +315,19 @@ public final class Guardrails implements GuardrailsMBean
         replicaDiskUsage.minNotifyIntervalInMs(minNotifyInterval);
     }
 
+    /**
+     * Guardrail on the minimum replication factor.
+     */
+    public static final MinThreshold minimumRF =
```

Comment:
Maybe this could be named `minimumReplicationFactor`, in line with the property names.